examples/prompting/Zero_shot_prompting.ipynb

```diff
@@ -0,0 +1,256 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    I go to this restaurant every week, i love it so much.
</pre><p>nit: capitalize i</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>